src/main/java/gregtech/common/items/armor/AdvancedJetpack.java

```diff
@@ -103,13 +102,9 @@ public float getFallDamageReduction() {
     }
 
     @SideOnly(Side.CLIENT)
-    public boolean isNeedDrawHUD() {
-        return true;
-    }
-
     @Override
     public void drawHUD(ItemStack item) {
```

Comment:
This method is basically the same as the `super` method in `Jetpack`. This method could probably be removed, or at least call super instead of re-adding a bunch of information.